cmd/kubeadm/app/apis/kubeadm/v1beta3/conversion.go

```diff
@@ -24,14 +24,17 @@ import (
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 )
 
+// Convert_kubeadm_InitConfiguration_To_v1beta3_InitConfiguration converts a private InitConfiguration to public InitConfgiuration.
```

Comment:
here is an example of a very naive, obvious comment. this applies to */conversion.go files in most cases.
in some cases we can include a reason on why the converter is needed, at least.
but it does not matter much.

Reply:
I was indeed wondering why this one was needed...

Reply:
i think it's because of missing fields in the peer type:
https://github.com/kubernetes/kubernetes/blob/6df1287be8bed0618e4bd23dc5cdb16779557369/cmd/kubeadm/app/apis/kubeadm/v1beta3/zz_generated.conversion.go#L618

https://github.com/kubernetes/kubernetes/blob/6df1287be8bed0618e4bd23dc5cdb16779557369/cmd/kubeadm/app/apis/kubeadm/v1beta3/zz_generated.conversion.go#L583

https://github.com/kubernetes/kubernetes/blob/6df1287be8bed0618e4bd23dc5cdb16779557369/cmd/kubeadm/app/apis/kubeadm/v1beta3/zz_generated.conversion.go#L585